audbcards/core/config.py

```diff
@@ -0,0 +1,19 @@
+class config:
+    r"""Get/set configuration values for the :mod:`audbcards` module.
+
+    Examples:
+        >>> config.CACHE_ROOT
+        '~/.cache/audbcards'
+        >>> config.CACHE_ROOT = "~/audbcards"
+        >>> config.CACHE_ROOT
+        '~/audbcards'
+
+    """
+
+    CACHE_ROOT = "~/.cache/audbcards"
+    r"""Default cache folder.
+
+    It can be overwritten
+    by the ``AUDBCARDS_CACHE_ROOT`` environment variable.
```

Comment:
In `audb`, it is always `CACHE_ROOT`. Here there is a slight asymmetry `AUDBCARDS_CACHE_ROOT` as an env variable and `CACHE_ROOT` when the variable lives inside the package.

Reply:
The asymmetry of `audbcards.config.CACHE_ROOT` and `AUDBCARDS_CACHE_ROOT` is the same as we have in `audb`, compare https://audeering.github.io/audb/caching.html#changing-cache-locations

Reply:
True. Misread. Apologies. Resolving this.